name_that_hash/hashes.py

```diff
@@ -1641,4 +1641,59 @@ class HashInfo:
                 )
         ],
     ),
+    Prototype(
+    regex=re.compile(r"\$bitcoin\$[0-9]{2,4}\$[a-fA-F0-9$]{250,350}", re.IGNORECASE),
+    modes=[
+        HashInfo(
+            name="Bitcoin / Litecoin",
+            hashcat=11300,
+            john="bitcoin",
+            extended=False,
+            description="Use Bitcoin2John.py to extract the hash for cracking."
```

Comment:
The description doesn't make sense anymore because the regex is for the converted hash. You should change it to something else.